server/database/schema.cql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS products (
+  productId int,
+  name text,
+  price decimal,
+  prime smallint,
+  imageUrl text,
+  numReviews int,
+  avgRating float,
+  category int
+);
+
+CREATE TABLE IF NOT EXISTS products ("productId" int, name text, price decimal, prime int, "imageUrl" text, "numReviews" int, "avgRating" float, category text, PRIMARY KEY ("productId", category)) WITH CLUSTERING ORDER BY (category ASC);
+
+dsbulk load --schema.keyspace related_items --schema.table products --connector.csv.url "/users/benjaminbarnett/Documents/Hack Reactor/sdc/rainforest-related-items.csv"
```

Comment:
Nice use of dsbulk load to load your csv file - I'll try it out as well. Also, you may be missing a newline at the end of the file.

Reply:
I did not actually run dsbulk from inside the .CQL file- that is deceptive. I was using the file to keep track of all necessary steps for my seeding.

I'm honestly not sure if .CQL files can be run by the Cassandra shell.